### ISSUE TITLE
Spinner button text not centered

### ISSUE THREAD

**Issue by AhmedMKamal** [michaeldoye/mat-progress-buttons#7]
Why spinner button's text isn't centered?

**Comment by michaeldoye** [comment]
How are you using the spinner button? Can you post a screenshot of this?

**Comment by AhmedMKamal** [comment]
> components.ts
```
export class LoginPageComponent {
  spinnerButtonOptions: ButtonOpts;
  constructor() {
  this.spinnerButtonOptions = {
      active: false,
      text: 'Login',
      spinnerSize: 20,
      raised: true,
      buttonColor: 'accent',
      spinnerColor: 'primary',
      mode: 'indeterminate',
      fullWidth: false,
      disabled: false
    };
  }
}
```
> components.html
```
<spinner-button [options]="spinnerButtonOptions"></spinner-button>
```

I'm using the following CSS to center text in the inactive mode.
```
.spinner-button {
  .mat-button-wrapper {
    span {
      margin-left: auto;
      margin-right: auto;
    }

    text-align: center !important;
  }
}
```

also, I added this class ```.spinner-button``` manually it's not there by default.

**Comment by alexmantaut** [comment]
I have the same problem. This is not visible when the text long, but it starts being noticeable on short texts such as 'Login'. Also, is there a way to set the size of the button?

**Comment by michaeldoye** [comment]
fixed in [7.0.4](https://github.com/michaeldoye/mat-progress-buttons/releases/tag/v7.0.4)
